endpoints/contacts.md

```diff
@@ -46,13 +46,20 @@
 
 ## Get Contacts
 
-* ```GET /contacts.json``` returns all contacts for the account
-* ```GET /groups/:group_id/contacts.json``` returns all contacts for specified group
+* ```GET /contacts.json``` returns a paginated list of contacts for the account
+* ```GET /groups/:group_id/contacts.json``` returns a paginated list of contacts for specified group
 
-Returns a ```200 OK``` with an array of contacts. (See [Contact](#contact) for JSON representation.) If you request a page of contacts that does not exist, a ```404 Not Found``` will be returned.
+Returns a ```200 OK``` with an array of contacts. (See [Contact](#contact) for JSON representation.) If you request a page of contacts that does not exist, a ```404 Not Found``` will be returned. (See [Pagination](https://github.com/outstand/api-docs#pagination))
 
 When specifying a group, if the group does not exist a ```404 Not Found``` will be returned.
 
+### Optional Query Params
+
+* `q` search query
```

Comment:
'search query' feels ambiguous.  Do we want to provide more info here?  Fulltext search query on which fields?